tests/integration/graphql/test_graphql.py

```diff
@@ -77,13 +77,33 @@ def test_id_only(req_type):
     '''
         % req_type
     )
-    print(response)
     assert 'data' in response
     assert 'docs' in response['data']
     assert len(response['data']['docs']) == 1
     assert set(response['data']['docs'][0].keys()) == {'id'}
 
 
+@pytest.mark.parametrize('req_type', ['mutation', 'query'])
+def test_data_list(req_type):
+    texts = 'abcd', 'efgh', 'ijkl'
+    response = graphql_query(
+        '''
+        %s {
+            docs(data: [{text: "%s"}, {text: "%s"}, {text: "%s"}]) { 
+                text
+            } 
+        }
+    '''
```

Comment:
can u use `f'{}` style better? the codebase is more used to that syntax

Reply:
I would love to, but f-strings don't seem to like strings that have `"` inside of them. Maybe there is a way, but couldn't get it to work.